.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,52 @@
+name: PR Check
+
+on: [pull_request]
+
+# When a new revision is pushed to a PR, cancel all in-progress CI runs for that
+# PR. See https://docs.github.com/en/actions/using-jobs/using-concurrency
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  go-lint:
+    name: Go Lint
+    runs-on: ubuntu-22.04
```

Comment:
🤔 I wonder about picking `ubuntu-22.04` rather than `ubuntu-latest` just because I worry I will forget to update it.

Reply:
That's a good thought. I'll adjust that momentarily.

Reply:
It is done. :+1: